client/controllers/events/curatedEvent.coffee

```diff
@@ -40,12 +79,12 @@ Template.curatedEvent.events
   'click .open-incident-form-in-details': (event, instance) ->
     data = instance.data
     Modal.show 'incidentModal',
-      articles: data.articles
-      userEventId: data.userEvent._id
+      articles: Articles.find()
+      userEventId: UserEvents.findOne(instance.data.userEventId)
```

Comment:
I think we can remove this? Looks to be old code and replaced by this: https://github.com/ecohealthalliance/eidr-connect/blob/ac8eeff38ee27022b06db37ceed3b2bf1748f9e1/client/controllers/incidentReports/addIncidentReport.coffee#L2-L7